views/export3.jade

```diff
@@ -3,34 +3,35 @@ extends layout
 block content
     include topnav
     h1 Preview
-    each songs in songs2d
+    //Note: songs2d = [{songs: [Song Schema that the user checked]}, {minLine: the smallest lyric line}]    
+    each songsObj in songs2d
         -var i = 0
         -var j = 0
-        -while (i < songs.length - 1 && j < songs.length - 1)
+        -while (i < songsObj.songs.length && j < songsObj.songs.length)
```

Comment:
before i put the minLine on the last index of array like [[obj, obj, minLine], [obj, minLine]], now i change it to be [{song: [obj], minLine: Integer}]